Jenkinsfile

```diff
@@ -66,6 +73,8 @@ pipeline {
         booleanParam name: 'BUILD_ALL_FLAVOURS', defaultValue: false, description: 'When selected all flavours are built and archived as artifacts that can be installed alongside other versions of the same APK.'
         string name: 'DEBUG_LABEL', defaultValue: '', description: 'For debugging when entered will be used as label to decide on which slaves the jobs will run.'
         string name: 'DOCKER_LABEL', defaultValue: '', description: 'When entered will be used as label for docker catrobat/catroid-android image to build'
+        string name: 'ADMOB_BANNER_APP_ID', defaultValue:'ca-app-pub-5619801003513533~9206664779', description: 'AdMob banner App id'
+        string name: 'ADMOB_BANNER_UNIT_ID', defaultValue:'ca-app-pub-5619801003513533/1184704529', description: 'AdMob banner unit id'
```

Comment:
Duplicate strings?

Reply:
nope, these are default values to be shown in the text fields